mapping/reactive-project-level-identification-preparation.csv

```diff
@@ -1,32 +1,32 @@
+Project officials and roles,Name and position of the highest authority in the procuring entity where the budget for this project is allocated (project owner),"From the `parties` array, get the `Organization` object whose role is 'publicAuthority'. If none exists yet:
 
 * Add a party
 * Set its `.name` to the name of the procuring entity
-* Set its `.role` to ‘procuringEntity’
+* Set its `.role` to 'procuringEntity'
 
-Add a `Person` object to the organization’s `.people` array and:
+Add a `Person` object to the organization's `.people` array and:
 
 * Set its `.id` incrementally
-* Set its `.name` to the project official’s name
-* Set its `.jobTitle` to the project official’s job title","Check the `/parties` array for `Organization` objects with 'procuringEntity' in `/roles`. Copy each object to the `parties` array in OC4IDS. If there is more than one procuring entity, do not set `.summary.tender.procuringEntity.name` and `.summary.tender.procuringEntity.id`; otherwise, set them to the procuring entity's `.name` and `.id`."
-Financial agreement,Loan or donation agreement with financing conditions.,[Add a project document](../common.md#add-a-project-document) and set `.documentType` to ‘financialAgreement’.,*Check the `/planning/documents` array for `Document` objects with `.documentType` set to ‘financialAgreement’ and copy each object to the `documents` array.*
-Procurement plan,Documentation that sets out the basis for this particular contracting process.,[Add a project document](../common.md#add-a-project-document) and set `.documentType` to ‘procurementPlan’.,*Check the `/planning/documents` array for `Document` objects with `.documentType` set to ‘procurementPlan’ and copy each object to the `documents` array.*
-Project approval decision,Additional details about the approval of the budget.,[Add a project document](../common.md#add-a-project-document) and set `.documentType` to ‘budgetApproval’.,*Check the `/planning/documents` array for `Document` objects with `.documentType` set to ‘budgetApproval’ and copy each object to the `documents` array.*
+* Set its `.name` to the project official's name
+* Set its `.jobTitle` to the project official's job title","Check the `/parties` array for `Organization` objects with 'procuringEntity' in `/roles`. Copy each object to the `parties` array in OC4IDS. If there is more than one procuring entity, do not set `.summary.tender.procuringEntity.name` and `.summary.tender.procuringEntity.id`; otherwise, set them to the procuring entity's `.name` and `.id`.","parties,parties/name,parties/roles,parties/people,parties/people/id,parties/people/name,parties/people/jobTitle"
```

Comment:
```suggestion
* Set its `.jobTitle` to the project official's job title","Check the `/parties` array for `Organization` objects with 'procuringEntity' in `/roles`. Copy each object to the `parties` array in OC4IDS. If there is more than one procuring entity, do not set `.summary.tender.procuringEntity.name` and `.summary.tender.procuringEntity.id`; otherwise, set them to the procuring entity's `.name` and `.id`.","parties,parties/name,parties/roles,parties/people,parties/people/id,parties/people/name,parties/people/jobTitle",parties/roles='procuringEntity'
```